### PR TITLE
Grpc/pass consistency level in tenants get

### PR DESCRIPTION
**Pull request by tsmith023**
### Review checklist

- [ ] Documentation has been updated, if necessary. Link to changed documentation:
- [ ] Chaos pipeline run or not necessary. Link to pipeline:
- [x] All new code is covered by tests where it is reasonable.
- [ ] Performance tests have been run or not necessary.

### CODE REVIEW

File: adapters/handlers/grpc/v1/tenants.go [hunk 1]
```diff
@@ -23,11 +23,12 @@ func (s *Service) tenantsGet(ctx context.Context, principal *models.Principal, r
 	if req.Collection == "" {
 		return nil, fmt.Errorf("missing collection %s", req.Collection)
 	}
+	isConsistent := req.GetConsistencyLevel() == pb.ConsistencyLevel_CONSISTENCY_LEVEL_ALL
 
 	var err error
 	var tenants []*models.Tenant
 	if req.Params == nil {
-		tenants, err = s.schemaManager.GetConsistentTenants(ctx, principal, req.Collection, req.IsConsistent, []string{})
+		tenants, err = s.schemaManager.GetConsistentTenants(ctx, principal, req.Collection, isConsistent, []string{})
```
Comment (reyreaud-l) [inline adapters/handlers/grpc/v1/tenants.go:31]: The consistent flag here isn't meant to reflect the consistency level when getting tenants (I.E it isn't ONE, QUORUM or ALL) it's meant to represent whether or not we want to query the leader for the information, or if we are fine with reading the local state on the follower which might eventually consistent.

File: grpc/proto/v1/tenants.proto [hunk 2]
```diff
@@ -16,7 +18,7 @@ enum TenantActivityStatus {
 
 message TenantsGetRequest {
   string collection = 1;
-  bool is_consistent = 2;
+  optional ConsistencyLevel consistency_level = 2;
```
Comment (reyreaud-l) [inline grpc/proto/v1/tenants.proto:21]: I would say considering the way it's used it makes sense to keep it a boolean. What do you think ? What was the reasoning behind changing this to a consistency level ?

**Review by reyreaud-l — COMMENTED** [review]
I want to understand why we believe this change to be necessary, to be sure we are not confusing consistency level with "consistent" for eventual consistency.

File: adapters/handlers/grpc/v1/tenants.go (hunk 1 above)
Comment (tsmith023) [reply adapters/handlers/grpc/v1/tenants.go:31]: @dirkkul and I had a convo on the Python PR about the origin of the `isConsistent` flag and its relation to the `ConsistencyLevel` [here](https://github.com/weaviate/weaviate-python-client/pull/1031#discussion_r1583414232). Would you say that the concepts of ONE, QUORUM, ALL are no longer applicable to eventual consistency? If so, we should look at a new way to represent this in the clients

File: grpc/proto/v1/tenants.proto (hunk 2 above)
Comment (tsmith023) [reply grpc/proto/v1/tenants.proto:21]: The reasoning is to align it with how the concept of consistency was used in previous versions. However, if this no longer makes sense generally (as above) then we should change the nomenclature that we use

**Comment by dirkkul** [comment]
closed after discussion, will remove consistency setting completely

**Pull request closed without merging**